docs/en/tutorials/useful_hook.md

```diff
@@ -0,0 +1,43 @@
+# Tutorial 13: Useful Hooks
+
+MMDetection and MMCV provide users with various useful hooks including log hooks, evaluation hooks, NumClassCheckHook and etc. . This tutorial mainly introduce function and usage of hooks in MMDetection\. For how to add custom hook to MMDetection, please read [customize_runtime](https://mmdetection.readthedocs.io/en/latest/tutorials/customize_runtime.html#customize-self-implemented-hooks),  and for hooks in MMCV, please read [docs in MMCV](https://github.com/open-mmlab/mmcv/blob/master/docs/en/understand_mmcv/runner.md).
```

Comment:
- 'NumClassCheckHook and etc. .' -> 'NumClassCheckHook, etc.'
- mainly introduce -> introduces the functionalities and usages of hooks implemented in MMDetection.
- We should add a brief example here to indicate how to add a custom hook, then refer readers to the doc for more details.
- and for hooks in MMCV -> For using hooks in MMCV, please read the API documentation in MMCV.